backend/mem/fileSystem.go

```diff
@@ -0,0 +1,49 @@
+package mem
+
+import (
+	"github.com/c2fo/vfs/v4"
+	"github.com/c2fo/vfs/v4/backend"
+	"github.com/c2fo/vfs/v4/utils"
+)
+
+//Scheme defines the filesystem type.
+const Scheme = "file"
```

Comment:
"file" is the scheme used for os files.
I suggest that Scheme should be "mem" or "memfs", such that when building a URI, like:
```
         foo://example.com:8042/over/there
         \_/   \______________/\_________/
          |           |            |
       scheme       volume        path
```
we would end up with:
`mem://myNamespace/some/path/to/file/`